src/main/java/io/confluent/connect/jdbc/TableMonitorThread.java

```diff
@@ -102,7 +104,7 @@ private boolean updateTables() {
     synchronized (db) {
       final List<String> tables;
       try {
-        tables = JdbcUtils.getTables(db);
+        tables = JdbcUtils.getTables(db,table_types);
```

Comment:
Isn't it easier to make the default setting to get all the types? Or is there a downside to that?